### DISCUSSION TITLE
HTTP URIs get rendered in dark red

### DISCUSSION THREAD

**Discussion by mfulton26** [github-linguist/linguist#5942] [Classification] [answered]
URIs in HTTP code blocks are rendered with a burgundy color (`#82071e`)

<table>
<tr>
<th>
source
</th>
<th>
rich
</th>
</tr>
<tr>
<td>

````markdown
```http
GET https://www.example.com/
```
````

</td>
<td>
<img src="https://user-images.githubusercontent.com/5588957/174350364-1853caf5-9228-41a8-b15f-c33fd6eab057.png" width="232">
</td>
</tr>
</table>

I could be mistaken but this leads me to believe that the URI isn't properly being matched or something as this same burgundy color (`#82071e`) is used elsewhere for "invalid" syntax

<table>
<caption>Example: JSON does not support comments but JSON5 does</caption>
<tr>
<th>
source
</th>
<th>
rich
</th>
</tr>
<tr>
<td>

````markdown
```json
// this is a comment
{ "answer": 42 }
```
````

</td>
<td>
<img src="https://user-images.githubusercontent.com/5588957/174350106-505b04b3-37be-48f0-9bba-6cd7c5082d58.png" width="175">
</td>
</tr>
<tr>
<td>

````markdown
```json5
// this is a comment
{ "answer": 42 }
```
````

</td>
<td>
<img src="https://user-images.githubusercontent.com/5588957/174348665-f280fd06-32ca-4b6e-a8da-fb6b198f73b8.png" width="175">
</td>
</tr>
</table>

I think it would be ideal for the URI to get rendered as either plain text, colored text (a hyperlink color?), or perhaps even rendered as a link (e.g. [`https://www.example.com/`](https://www.example.com/)).

Is this a correct understanding or are HTTP URIs supposed to be rendered burgundy for some reason? Please advise. Thank you.

**Comment by lildude** [comment, marked as answer]
Your question is better suited in the [HTTP grammar repo](https://github.com/samsalisbury/Sublime-HTTP) that is responsible for the syntax highlighting, not here, as this is a grammar issue.

**Reply by mfulton26** [reply]
I've asked there (see https://github.com/samsalisbury/Sublime-HTTP/issues/7). I'm not sure how active that project is though…
